src/components/application/connectors/coinbase-wallet-wagmi-connector.ts

```diff
@@ -0,0 +1,279 @@
+import type {
```

Comment:
Added this incase we have existing issues with the connector. This is wagmi's implementation modified for coinbase/wallet-sdk v4.2.1

Reply:
Thanks!